README.md

```diff
@@ -49,54 +49,83 @@ Another option is to open it with Control-click: it'll immediately register the
+
 - [ ] smaller screen support (UI refactor)
 
 ### tools
 
 - [ ] App automated end to end tests
+
 - [ ] more technical documentation (install & release process notably)
 
 ### release
 
 - [ ] github page
+
 - [ ] usage statistics
+
 - [ ] references
+
   - [ ] Electron's [app list](https://www.electronjs.org/apps)
+
   - [ ] Svelte's [showcase](https://svelte-community.netlify.app/showcase)
+
   - [ ] [Snap store](https://snapcraft.io/)
 
 ### Bugs and known issues
 
-1. Playlist are not updated on tracks removal/changes
+1. Playlist models are not updated on tracks removal
+
 1. Undetected live changes: remove tracks and re-add them. This is a linux-only issue with chokidar
-   - https://github.com/paulmillr/chokidar/issues/917
-   - https://github.com/paulmillr/chokidar/issues/591
-1. Files renamed or moved to other watched folders are removed and re-added. This is a limitation with chokidar
-   - https://github.com/paulmillr/chokidar/issues/303
+
+   - [Issue #917](https://github.com/paulmillr/chokidar/issues/917)
+
+   - [Issue #591](https://github.com/paulmillr/chokidar/issues/591)
+
 1. When loading new folders, enqueuing or going to album details will give incomplete results. Going back and forth won't load new data
+
 1. Security: clean html in artist/album names (wrapWithRefs returns injectable markup)
-1. AppImage, when used with AppImageLauncher, fail to auto update: https://github.com/electron-userland/electron-builder/issues/4046#issuecomment-670367840
+
+1. AppImage, when used with AppImageLauncher, [fail to auto update](https://github.com/electron-userland/electron-builder/issues/4046#issuecomment-670367840)
+
 1. If we knew current position in browser history, then we could disabled navigation button accordingly
+
 1. Page navigation: use:link doesn't work in tests and raise Svelte warning. a.href is fine
+
 1. Disklist/TrackTable dropdown does not consider scroll position (in storybook only)
+
 1. Testing input: fireEvent.change, input or keyUp does not trigger svelte's bind:value on input
+
 1. The test suite is becoming brittle
+
    1. `Media service › triggerAlbumsEnrichment › saves first returned cover for album`
+
       ```shell
       > 839 |       expect(await fs.readFile(savedAlbums[0].media, 'utf8')).toEqual(
           |                                                               ^
       ```
+
    1. `Media service › triggerAlbumsEnrichment › retries album with no cover but at least one restriced provided`
       Is a 1ms difference in expected `processedEpoch`
+
    1. `TracksQueue component › given a list of tracks › clears tracks queue`
       The list items are still visible after clear (probably because of the animation)
+
    1. `AddToPlaylist component › given some playlists › saves new playlist with all tracks`
       The dropdown menu is still visible (probably because of the animation)
 
+1. The Media test do not pass on Windows: nock is not giving recorded bodies
```

Comment:
Codacy found an issue: [[ordered-list-marker-value] Marker should be `11`, was `1`](https://app.codacy.com/gh/feugy/melodie/pullRequest?prid=6368165)